controllers/chat/getChannelDetail.js

```diff
@@ -53,8 +53,14 @@ const getChannelDetail = async (req, res) => {
     });
   }
 
+  const shouldBetterChannelMembersUpdated = channel_type !== CHANNEL_TYPE.GROUP;
+
   const {betterChannelMember, betterChannelMemberObject, updatedChannel} =
-    await BetterSocialCore.chat.updateBetterChannelMembers(channel, createdChannel, true);
+    await BetterSocialCore.chat.updateBetterChannelMembers(
+      channel,
+      createdChannel,
+      shouldBetterChannelMembersUpdated
+    );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic to decide whether to update better channel members has been changed. Now it checks if the channel type is not a group before updating. This change aligns with the PR title and seems to be a fix for an issue where better channel members were being updated unnecessarily for group channels.
```diff
-  const {betterChannelMember, betterChannelMemberObject, updatedChannel} =
-   await BetterSocialCore.chat.updateBetterChannelMembers(channel, createdChannel, true);
+  const shouldBetterChannelMembersUpdated = channel_type !== CHANNEL_TYPE.GROUP;
+  
+  const {betterChannelMember, betterChannelMemberObject, updatedChannel} =
+   await BetterSocialCore.chat.updateBetterChannelMembers(
+     channel,
+     createdChannel,
+     shouldBetterChannelMembersUpdated
+   );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->